src/routes/read/[slug].svelte

```diff
@@ -88,58 +89,93 @@
 
 <MetaTags {insight} />
 
-<div class="insight">
-  {#if process.browser && $session.isDesktop}
-    <FixedControls {insight} {link} {hidden} {isAuthor} {isDraft} />
-    {#if projectData && isPaywalled === false}
-      <Assets {insight} {projectData} />
+<div class="content relative">
```

Comment:
Root `_layout` should have bottom padding/margin to compensate for a mobile nav. This `div` is redundant